README.md

```diff
@@ -17,34 +17,74 @@ Example application for handling ShipEngine Elements authentication token reques
 
 1. Install dependencies
 
+Install with npm
+
 ```bash
 npm install
 ```
 
+Installl with yarn
+
 ```bash
 yarn
 ```
 
 2a. Start the application
 
+Start with npm
+
 ```bash
 npm start
 ```
 
+Start with yarn
+
 ```bash
 yarn start
 ```
 
 2b. Start application for development
 
+Start with npm in dev mode (watch mode)
+
 ```bash
 npm dev
 ```
 
+Start with yarn in dev mode (watch mode)
+
 ```bash
 yarn dev
 ```
```

Comment:
A little chunky here. These can be consolidated into either npm or yarn given that the scripts are the same and then could add a blurb stating to replace with npm/yarn for preference if you think that's necessary.